spec/models/family_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Family, type: :model do
+  let(:user) { create(:user) }
+  let(:family) { create(:family) }
+
+  context 'validation' do
+    %i[user_id].each do |field|
+      it { is_expected.to validate_presence_of(:"#{field}") }
+    end
+  end
+
+  context 'assotiation' do
+    it { is_expected.to have_many(:users) }
+  end
+
+  context 'creating' do
+    before { user }
+    it 'be valid with valid attributes' do
+      expect(family).to be_valid
+    end
```

Comment:
Useless example. You just create a valid object using factory and checking it's valid.
`expect(true).to be true` is pretty similar.
Correct example:
```ruby
let(:family) { build(:family, field: nil) }
expect(family).to be_invalid
```
Anyway, use [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers) for most model tests

Reply:
got it